cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/GateChange.java

```diff
@@ -40,6 +44,9 @@ public class GateChange {
     @JsonProperty("notes")
     private String notes = null;
 
+    @JsonProperty("type")
```

Comment:
Is this a default type for this? But it can be changed?

Should GetSetting also have similar behavior with a default type?

Reply:
This is part of the sub-typing setup for serialization/deserialization. I was able to remove the getter/setter without losing functionality.

I also added a default to GateSetting for consistency.